docs/spark/how-to-guides/hdinsight-notebook-installation.md

```diff
@@ -0,0 +1,123 @@
+
+1.2 Select **Spark2**, and then select **LIVY FOR SPARK2 SERVER**.
+
+<img src="../media/hdi-spark-notebooks/select-livyserver.png" alt="StopLivyServerImage" width="800"/>
+
+1.3 Select **hn0... host**, stop **Livy for Spark2 Server** if started. When prompted, click **OK** to proceed.
+
+- Select hn0 as shown below.
+<img src="../media/hdi-spark-notebooks/select-host.png" alt="StopLivyServerImage" width="800"/>
+
+- Stop Livy for Spark2 Server.
+<img src="../media/hdi-spark-notebooks/stop-server.png" alt="StopLivyServerImage" width="800"/>
+
+1.4 Please follow the same step for hn1... host.
+
+### Step 2. Submit HDInsight Script Action
+Return to your HDI cluster in the Azure Portal, and select **Script actions** from the options on the left. You'll submit one script action to deploy SparkDotnet REPL on HDInsight Spark cluster.
+
+Submit Script Action on both Head node and Worker node.
+Run `install-interactive-notebook.sh` on the cluster using [HDInsight Script Actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux):
+
+* Script type: Custom
+* Name: Install Sparkdotnet Interactive Notebook Experience (or anything that is descriptive)
+* Bash script URI: The URI to which you uploaded `install-interactive-notebook.sh`. Create a new file named **install-interactive-notebook.sh** in your local computer, and paste the [install-interactive-notebook.sh contents](https://raw.githubusercontent.com/dotnet/spark/master/deployment/HDI-Spark/Notebooks/install-interactive-notebook.sh). Then, upload the script to a [URI](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux#understand-script-actions) that's accessible from the HDInsight cluster. (e.g. `https://<my storage account>.blob.core.windows.net/<my container>/<some dir>/install-interactive-notebook.sh`)
+* Node type(s): Head and Worker
+* Parameters: Sparkdotnet version. For example, if you want to install Sparkdotnet version 0.6.0 then it would be `0.6.0`.
+
+The following captures the setting for a HDInsight Script Action on Head node and Worker node:
+
+<img src="../media/hdi-spark-notebooks/install-notebook-scriptaction.png" alt="ScriptActionImage" width="500"/>
+
+After you see green checkmarks next to the status of both script actions, you can move to the next step.
+
+### Step 3. Start Livy Server
+Please follow the above [Step 1](#step-1-stop-livy-server) to **Start** Livy for Spark2 Server.
+
+### Step 4. Set up Spark Default Configs
+4.1 From the portal, select **Overview**, and then select **Ambari home**. If prompted, enter the cluster login credentials for the cluster.
+
+4.2 Select **Spark2**, and then Select **CONFIGS**, then click on **Custom spark2-defaults**.
+
+<img src="../media/hdi-spark-notebooks/spark-configs.png" alt="SetConfigsImage" width="800"/>
+
+4.3 Click on **Add Property...** to add Spark default settings.
```

Comment:
Do you think we can/should add a screenshot of **Add Property**? Since we've shown images for most of the other steps, and I think they've been really helpful.

Reply:
I have added the screenshot for **Add Property**.